Python/Scrap_Restaurants_using_ZomatoAPI/main.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import csv
+
+file_name = input(
+    "Enter the file name to save Restaurants List with extension name .csv : ")
+with open(file_name, "a", newline='') as fp:
+
+    wr = csv.writer(fp, dialect='excel')
+    # Change City with &q=""
+    url = "https://developers.zomato.com/api/v2.1/search?entity_id=4&entity_type=city&q=Mumbai&start=1"
+    header = {"Accept": "application/json", "user-key": "API-KEY",
+              "User-agent": "curl/7.43.0"}  # Replace API-KEY with your Zomato API, API Limit: 1000 Calls/day
```

Comment:
Please have the API Key in a `.env` file and use a module like dotenv to import them in your Python Code

Reply:
I need help regarding this, pls help

Reply:
is it needed? i am unable to implement this part.